libvast/include/vast/query_queue.hpp

```diff
@@ -55,9 +55,9 @@ class query_queue {
     std::vector<uuid> queries;
     bool erased = false;
 
-    friend auto operator<=>(const entry& lhs, const entry& rhs) {
-      return lhs.priority <=> rhs.priority;
-    }
+    friend bool operator<(const entry& lhs, const entry& rhs) noexcept;
+    friend bool operator==(const entry& lhs, const uuid& rhs) noexcept;
+    friend bool operator==(const uuid& lhs, const entry& rhs) noexcept;
```

Comment:
Only one of these should be needed in C++20, as the [primary comparison operators can be reversed](https://brevzin.github.io/c++/2019/07/28/comparisons-cpp20/).